.github/workflows/release.yml

```diff
@@ -13,11 +13,17 @@ jobs:
     steps:
       - name: Checkout
         uses: actions/checkout@v3
-      - name: Set up Python
-        uses: actions/setup-python@v4
-        with:
-          python-version: '3.11' 
-          cache: poetry
+      - name: Install Python
+        run: apt-get update && apt-get install -y python3-dev python3-venv
```

Comment:
Can we use python3.10-dev here and everywhere else to avoid potential breaking on new python version being released?